### ISSUE TITLE
Not meching the parameter setting what is written in paper

### ISSUE THREAD

**Issue by Dev100123** [tqwei05/PlantSeg#10]
I look into the config file and check every model's parameter, which you mention on the run.sh file.

In the paper, you said you use the SGD optimiser, 0.001 learning rate, 0.9 momentum, used 16 batch size to all models, but i dont find any matching with it in the config. I saw AdamW, 0.0006, batch size 4, parameter many models, and many models for plantseg115, you used different _base_. schedules python files😅

Please, can you kindly explain all of this?

**Comment by tqwei05** [comment]
We tried different training settings so the released configs may differ. Please feel free to adjust the configs.

**Comment by Dev100123** [comment]
I just wanna ask you how many epochs you used, 40000 or 160000? optimiser SGD or AdamW?

**Comment by tqwei05** [comment]
> I just wanna ask you how many epochs you used, 40000 or 160000? optimiser SGD or AdamW?

160000 with SGD.

**Comment by Dev100123** [comment]
I just wanna tell you, I am training my own model with these configs. Can you tell its ok for this dataset, right? I got this from your model "segnext_mscan-t_1xb16-adamw-40k_plantseg115-512x512.py", I just changed the optimizer AdamW to SGD and the epochs from 40000 to 160000. This is configs:

_base_ = [
    '/lustre/home/rchandraghosh/plantseg-latext/configs/_base_/default_runtime.py', '/lustre/home/rchandraghosh/plantseg-latext/configs/_base_/schedules/schedule_160k.py',
    '/lustre/home/rchandraghosh/plantseg-latext/configs/_base_/datasets/plantseg115.py'
]

import torch
torch.set_num_threads(1)
torch.set_num_interop_threads(1)

data_preprocessor = dict(
    type='SegDataPreProcessor',
    mean=[123.675, 116.28, 103.53],
    std=[58.395, 57.12, 57.375],
    bgr_to_rgb=True,
    pad_val=0,
    seg_pad_val=255,
    size=(512, 512),
    test_cfg=dict(size_divisor=32))

# optimizer
optim_wrapper = dict(
    _delete_=True,
    type='OptimWrapper',
    optimizer=dict(
        type='SGD', lr=0.01, momentum=0.9, weight_decay=0.0005),
    paramwise_cfg=dict(
        custom_keys={
            'pos_block': dict(decay_mult=0.),
            'norm': dict(decay_mult=0.),
            'head': dict(lr_mult=10.)
        }))
# learning policy
param_scheduler = [
    dict(
        type='LinearLR', start_factor=1e-6, by_epoch=False, begin=0, end=1500),
    dict(
        type='PolyLR',
        power=1.0,
        begin=1500,
        end=160000,
        eta_min=0.0,
        by_epoch=False,
    )
]

**Comment by tqwei05** [comment]
This config is based on work from a couple of years ago, so some details are no longer very clear to me. Your settings look reasonable, and please feel free to adjust them based on your experiments if needed.
